### PR TITLE
Fred review

### PR DESCRIPTION
**Pull request by Go-Fred**
Add some screenshots and more info.

### CODE REVIEW

**Comment by Go-Fred** [comment]
@spasiu

File: README.md [hunk 1]
```diff
@@ -2,18 +2,23 @@
 
 ## Get started
 
-For a detailed guide, see the [Smooch API Quickstart](https://docs.smooch.io/docs/api-quickstart)
+For a detailed guide, see the [Smooch API Quickstart](https://docs.smooch.io/docs/api-quickstart) and [our API QuickStart video tutorial](https://vimeo.com/198071401"):
```
Comment (jpjoyal) [inline README.md]: link is 404

File: README.md [hunk 2]
```diff
@@ -2,18 +2,23 @@
...
+
+<a href="https://vimeo.com/198071401" target="_blank"><img src="https://i.vimeocdn.com/video/610957143.webp?mw=1800&mh=1013&q=70"
+alt="VIMEO" width="240" border="10" /></a>
 
 ### Node.js
 
 1. Clone the repository
 2. Go to the _nodejs_ subdirectory
 3. Update index.js to include your secret key and key ID from your [Smooch](app.smooch.io) settings
+![Smooch App settings](http://i.imgur.com/9O7h9CM.png)
```
Comment (jpjoyal) [inline README.md]: There's a black bar on the side, and its now outdated

**Comment by jpjoyal** [comment]
rest looks good to me.

**Comment by Go-Fred** [comment]
@jpjoyal fixed! thanks.

**Comment by dannytranlx** [comment]
What's this, merge it!

**Pull request merged**